web/js/holidayManagement.js

```diff
@@ -17,6 +17,37 @@
  * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+function formatDate(date) {
+    // Receives a date object and returns a string in the format YYYY-MM-DD.
+    // The subtract below is to discard timezone.
+    return new Date(date.getTime() - (date.getTimezoneOffset() * 60000))
+        .toISOString()
+        .split("T")[0];
+}
+
+function deleteRangeAndDays(ranges, day, days) {
```

Comment:
Sorry, I don't understand the purpose of the `days` parameter here. Could you please add some comments? A short line about the purpose of the function, its inputs and outputs would be great.